Assignment 6/Ques3.2.js

```diff
@@ -0,0 +1,40 @@
+// 3.2 Create a class Student that inherits from the Person class and has a property studies. Override the greeting() method to include information about what the student is studying.
+
+class Person{
+    name;
+    age;
+    gender;
+    interest;
+    constructor(name,age,gender,interest)
+    {
+       this.name=name;
+       this.age=age;
+       this.gender=gender;
+       this.interest=interest;
+    }
+    greeting()
+    {
+       return "Hi my name is : "+this.name+"\nAge is "+this.age+"\nGender: "+this.gender+"."+"\nInterest: "+this.gender;
+    } 
+    farewell()
+    {
+       return "Goodby "+ this.name;
+    }
+   }
+   
+ class Student extends Person{
+    study;
+    constructor(name,age,gender,interest,study)
+    {
+        super(name,age,gender,interest);
+        this.study=study;
+    }
+    greeting()
+    {
```

Comment:
Try using the `greeting()` method from the `Person` class using `super.greeting()` to display attributes like name, age, gender, and interest. and add additional information about the student's studies in this greeting method. There is no need to rewrite code if it's already defined in the parent class.